packages/cache/src/options.ts

```diff
@@ -101,7 +101,7 @@ export function getDownloadOptions(copy?: DownloadOptions): DownloadOptions {
     useAzureSdk: true,
     downloadConcurrency: 8,
     timeoutInMs: 30000,
-    segmentTimeoutInMs: 3600000,
+    segmentTimeoutInMs: 600000,
```

Comment:
These were not updated to match:
- https://github.com/actions/toolkit/blob/457303960f03375db6f033e214b9f90d79c3fe5c/packages/cache/src/options.ts#L50-L55
- https://github.com/actions/toolkit/blob/457303960f03375db6f033e214b9f90d79c3fe5c/packages/cache/src/internal/downloadUtils.ts#L267

and README was updated in https://github.com/actions/toolkit/pull/1367 but RELEASES wasn't https://github.com/actions/toolkit/blob/457303960f03375db6f033e214b9f90d79c3fe5c/packages/cache/RELEASES.md?plain=1#L160-L162